crdbx/config.go

```diff
@@ -62,6 +65,34 @@ func (c Config) GetURI() string {
 	return u.String()
 }
 
+// MustRegisterArgs registers the arguments for the database connection.
+// Use in conjunction with MustViperFlags to ensure default values are set.
+func MustRegisterArgs(v *viper.Viper, flags *pflag.FlagSet) {
```

Comment:
since the `MustRegisterFlags` function already takes a `viper.Viper` instance and a flagset, why not just do the flag bindings there?

Reply:
I can rename `MustViperFlags` to `MustRegisterFlags` and merge the two functions.

Reply:
I've merged `MustRegisterArgs` into `MustViperFlags`